### ISSUE TITLE
Unable to run 

### ISSUE THREAD

**Issue by suhana960** [kithiyonjesus/RXCE#3]
Please provide installation commands

**Comment by rudrapatil122** [comment]
Hello,, bro command mili kya tume?

**Comment by MaheshDhanawade** [comment]
I also want code.

**Comment by thevectorengineer** [comment]
anyone get please inform

**Comment by rudrapatil122** [comment]
@thevectorvirus  anybody want rxce colour prediction ..msg me Teligram my I'd.@rxcewin123

**Comment by rudrapatil122** [comment]
@MaheshDhanawade anybody want rxce colour prediction ..msg me Teligram my I'd.@rxcewin123

**Comment by rudrapatil122** [comment]
@suhana960 anybody want rxce colour prediction ..msg me Teligram my I'd.@rxcewin123

**Comment by Mukesh-pu** [comment]
@rudrapatil122  teligram id kam nahi kar ri hai sir

**Comment by 93aditya** [comment]
Plz give me hack code i lost 300k

**Comment by rudrapatil122** [comment]
My teligram ***@***.***

Msg me fast..

**Comment by rudrapatil122** [comment]
> @rudrapatil122 teligram id kam nahi kar ri hai sir

Anybody want rxce hack msg me Teligram 👉@Royalboy546

💯 Sure hack❤️

**Comment by rudrapatil122** [comment]
> Please provide installation commands

Anybody want rxce hack msg me Teligram 👉@Royalboy546

💯 Sure hack❤️

**Comment by SmileyKomal** [comment]
> Hello,, bro command mili kya tume?

If anyone wants rxce hack
So please contact me
On that no.
9990327256
Please massage me fast on whatsapp 
Not on telegram 
It's 101% working so please come fast. 
And I will give you that hack.

**Comment by SmileyKomal** [comment]
> Plz give me hack code i lost 300k

If anyone wants rxce hack
So please contact me
On that no.
9990327256
Please massage me fast on whatsapp 
Not on telegram 
It's 101% working so please come fast. 
And I will give you that hack.

**Comment by SmileyKomal** [comment]
> Hello,, bro command mili kya tume?

If anyone wants rxce hack
So please contact me
On that no.
9990327256
Please massage me fast on whatsapp 
Not on telegram 
It's 101% working so please come fast. 
And I will give you that hack.

**Comment by SmileyKomal** [comment]
> Please provide installation commands

If anyone wants rxce hack
So please contact me
On that no.
9990327256
Please massage me fast on whatsapp 
Not on telegram 
It's 101% working so please come fast. 
And I will give you that hack.

**Comment by Sameer1580** [comment]
Jiske pass rxce hack hai muze msg karo.. teligram id👉@Sameer12689

**Comment by SmileyKomal** [comment]
RXCE hack:-
I have rxce hack those who want please contact me on that no.
9990327256
Please come fast because only 3 copy of that hack are left so come fast

**Comment by SmileyKomal** [comment]
> Please provide installation commands

RXCE hack:-
I have rxce hack those who want please contact me on that no.
9990327256
Please come fast because only 3 copy of that hack are left so come fast

**Comment by SmileyKomal** [comment]
> I also want code.

RXCE hack:-
I have rxce hack those who want please contact me on that no.
9990327256
Please come fast because only 3 copy of that hack are left so come fast

**Comment by SmileyKomal** [comment]
> Plz give me hack code i lost 300k

RXCE hack:-
I have rxce hack those who want please contact me on that no.
9990327256
Please come fast because only 3 copy of that hack are left so come fast

**Comment by SmileyKomal** [comment]
> Jiske pass rxce hack hai muze msg karo.. teligram id👉@Sameer12689

RXCE hack:-
I have rxce hack those who want please contact me on that no.
9990327256
Please come fast because only 3 copy of that hack are left so come fast

**Comment by 93aditya** [comment]
U have commands ??

**Comment by 93aditya** [comment]
Can u give me commands on termux

**Comment by SmileyKomal** [comment]
Yes, I have

**Comment by prem2904** [comment]
Accha @SmileyKomal  didi g... Agr aapke pass hack h to aap dusro ko q de re ho... Or agr help krne k leye de re ho.. To 5k ki demand q kr re ho..?? Oye  @93aditya same question puchna didi se.. Paise dene se phle..

**Comment by 93aditya** [comment]
Han.... Ap hame command dedo agar ye work kiya to ham apko 5k after winning de denge it's my promise

**Comment by prem2904** [comment]
@93aditya Accha beta tm hi tez ho.. Tm hack le kr bhar gya to... Or promise tod deya to.. Didi ko bhi to kuch security chaheye... Chalo @SmileyKomal didi aap ek kaam kro.. Aap zoom ya kisi bhi virtual aap k through.. Hame working hack dekha do.. Fir paise lelo.. Etna to hame bhi security ka hak hai na didi....

**Comment by prem2904** [comment]
@93aditya  Tu likh ke lelo... Didi reply nhi kregi...

**Comment by SmileyKomal** [comment]
Okay ap whatsapp kro mujhe 9990327256 es no pr

**Comment by prem2904** [comment]
Ooo hlo Mr didi g... WhatsApp q.. Whatsapp pe.. Fake video.. Fake SS dekhaoge.. @premium786 ese kholo bhut fake melenge.. Thik hai.. To gyan kam do.. Or zoom/google meet/anydesk ya jisme aap comfortable ho.. Uska link bhejo.. Or screen share kr k deekhao.. Yaha sb koi dekhenge.. Zada sell hoga.. Aapke hack ka.. Paise hi paise.. Honge.. Or bhi mere bhut bhai log hai jinhe hack ki zarurat hai... Or aap 5k ki baat krte ho.. Sb aapko advance me 10k denge... Or kaam pura hone k baad 50k jaao aish kro... Or agr koi nhi deya.. To mai dunga..sb ke badle.. Ab khush.. Chalo link bhejo

**Comment by prem2904** [comment]
@SmileyKomal Are ky hua didi... Aapne to kuch reply hi nhi ketta...

**Comment by 93aditya** [comment]
@premium786 fake h kya bhai ??

**Comment by 93aditya** [comment]
Ha prem ki baat sahi h kuch esa live demo do jis se ham sure ho jaae ki ha apka hck kaam karta h

**Comment by prem2904** [comment]
@SmileyKomal  are ky hua yaar didi aap kuch keh hi nhi re ho... Ek dum se chup q hogye ho... Maine to 7 customers bhi dhund leye hai.. Aapke hack lene k leye... Bolo kuch

**Comment by 93aditya** [comment]
@prem2904  bhai @premium786 fake h kya ? 
  And ek or hai @htr_tool vo bhi fake h kya ?

**Comment by prem2904** [comment]
Yup premium bhi fake hai... Or jo htr tool hai... Maine tahmid rayat se khud baat keya hai.. Usne kaha ki.. Usne aisa koi tool banaya hi nhi hai.. Or usne ye bhi kaha ki esse banana naammumkin hai.. Qki rxce ka color manually bhi change keya jata hai.. Or ye foto bhi foto bhi edited hai... Or wo hack wala video bhi....![Screenshot_2021-07-02-14-41-26-097_org.telegram.messenger.jpg](https://user-images.githubusercontent.com/64469877/133685345-8dd3a95a-d914-43d5-a8d3-ed2336b0fac7.jpg)

**Comment by prem2904** [comment]
Dekha na didi bhi kuch reply nhi ke.... Proof k naam aate hi didi ko saap sungh leya

**Comment by 93aditya** [comment]
@prem2904  ok bhai thanks yaar.... Bacha liya tune jo market bahut saare log hack ke naam pe paros rahe h sab fake h....... Premium786 to daily ss dalta h yaar gajab edit karta h banda ... Koi bhi jhanse me aa jae

**Comment by 93aditya** [comment]
To apne rayat se pucha nHi ki fake kyu banaya ye sab

**Comment by prem2904** [comment]
@93aditya fraud krne k leye simple...

**Comment by 93aditya** [comment]
To fir vo khule aam kyu bol raha h ki ye sab fake h or mujhe fraud karna h

**Comment by prem2904** [comment]
Ye to whi jaane.. Wo fake chats bana ra hai.. Proof ke tour pe.. Or fake bet amount dekha ra hai har cheez fake hai... Tune premium wale group me SS dekha... Wo roz nhi daalta hai... 2-3din pe daalta hai.. Or in dino me fake chats tyar krta hai.. 😂😂

**Comment by 93aditya** [comment]
I am agree with you bro .... But jesa ki aap bata rahe h ho ye bnde fraud karne ke liye itni mehnat kar rahe h... Iske bawjud ye apko ese hi openly bol diya ki ye sab ham fake bana rahe h samjh nahi aaya bhai
